docs/agent-development.md

```diff
@@ -334,7 +336,64 @@ In addition to each agent having a "native" API for instrumentation, they also i
 
 ### Distributed Tracing
 
-We implement the [W3C Trace Context](https://w3c.github.io/trace-context/) standard, with the exceptions that we currently use the header name "elastic-apm-traceparent" and we do not currently support "tracestate". Both of these exceptions will be removed as Trace Context nears finalisation; we are tracking this in https://github.com/elastic/apm/issues/71.
+We implement the W3C standards, both for HTTP headers and binary fields.
+
+#### Http Headers
+
+Our implementation relies on the [W3C Trace Context](https://www.w3.org/TR/trace-context-1/) standard. Until this standard became final
+, we used the header name `elastic-apm-traceparent` and we did not support `tracestate`. Soon after the standard became official, we
```

Comment:
I may be being pedantic, but can we move this comma to the end of the previous line? (Or move `final` to this line.) This also prevents an extra space before the comma in the final render.